packages/sdk/wallets/wallet-walletconnect/test/in-memory-wallet.ts

```diff
@@ -0,0 +1,145 @@
+import { CeloTx } from '@celo/connect'
+import { Address, newKit } from '@celo/contractkit'
+import { toChecksumAddress } from '@celo/utils/lib/address'
+import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
+import WalletConnect, { CLIENT_EVENTS } from '@walletconnect/client'
+import { PairingTypes, SessionTypes } from '@walletconnect/types'
+import { SupportedMethods } from '../src/types'
+
+const privateKey = '04f9d516be49bb44346ca040bdd2736d486bca868693c74d51d274ad92f61976'
+const kit = newKit('https://alfajores-forno.celo-testnet.org')
+kit.addAccount(privateKey)
+const wallet = kit.getWallet()!
+const [account] = wallet.getAccounts()
+
+export const testPrivateKey = privateKey
+export const testAddress = toChecksumAddress(account)
+
+// personal_sign is the one RPC that has [payload, from] rather
+// than [from, payload]
+function parsePersonalSign(req: any): { from: string; payload: string } {
+  const [payload, from] = req.payload.params
+  return { from, payload }
+}
+function parseSignTypedData(req: any): { from: string; payload: EIP712TypedData } {
+  const [from, payload] = req.payload.params
+  return { from, payload: JSON.parse(payload) }
+}
+function parseSignTransaction(req: any): CeloTx {
+  return req.payload.params
+}
+function parseComputeSharedSecret(req: any): { from: Address; publicKey: string } {
+  const [from, publicKey] = req.payload.params
+  return { from, publicKey }
+}
+function parseDecrypt(req: any): { from: string; payload: Buffer } {
+  const [from, payload] = req.payload.params
+  console.log(req.payload)
+  return { from, payload: Buffer.from(payload, 'hex') }
+}
+
+export function getTestWallet() {
+  let client: WalletConnect
+  let sessionTopic: string
+
+  function onSessionProposal(proposal: SessionTypes.Proposal) {
```

Comment:
Just a JS style tip though definitely no big deal: it's common to use arrow functions when defining functions inline (i.e. in another function). Partly for style and partly for how they bind `this` better.